sdks/python/apache_beam/io/vcfio.py

```diff
@@ -179,9 +179,6 @@ def __le__(self, other):
 
     return self < other or self == other
 
-  def __ne__(self, other):
```

Comment:
This method was defined twice.  I recommend using your favorite git UI to go through the commits one by one, because I give some context for each change in the commit message.